RedditSharp/Things/LiveUpdateEvent.cs

```diff
@@ -545,7 +539,7 @@ private void CommonInit(Reddit reddit, JToken json, IWebAgent webAgent)
             WebAgent = webAgent;
         }
 
-        private void SimpleAction(string url)
+        private new void SimpleAction(string url)
```

Comment:
Actually, that can't be done and when I try it visual studio complains with this:

```'LiveUpdateEvent.SimpleAction(string)': cannot change access modifiers when overriding 'protected' inherited member 'Thing.SimpleAction(string)'```

and

```LiveUpdateEvent.SimpleAction(string)': virtual or abstract members cannot be private```

So new is probably the only way we can go about that, as it's a private method and really has no reason to be used elsewhere as it's unique to that Thing type